client/app/scripts/liveblog-edit/module.js

```diff
@@ -312,7 +312,6 @@ define([
                 modal.confirm(gettext('Are you sure you want to remove the blog image?')).then(function() {
                     deregisterPreventer();
                     vm.newBlog.picture_url = null;
-                    vm.newBlog.picture = null;
```

Comment:
Why stopping to remove the picture reference ?
the field still exists in `blog.py`
`'picture': Resource.rel('archive', embeddable=True, nullable=True, type='string'),`